apps/happy-blocks/block-library/pricing-plans/hooks/pricing-plans.ts

```diff
@@ -61,6 +62,7 @@ const usePricingPlans = () => {
 				const response = await fetch( PLANS_API_URL );
 				const data = await response.json();
 				setPlans( parsePlans( data ) );
+				i18n.addTranslations( getLocaleData( 'happy-blocks' ) );
```

Comment:
Tested and it works good.

I've worked on a [slightly different implementation](https://github.com/Automattic/wp-calypso/commit/c600c7d92e415734a606b4fbd625ae7965c803e0), that uses essentially the same approach, but is behind a custom hook, intended to be used at the root component of each entry point that includes i18n-calypso strings (at this moment, it's the **pricing-plans** and **universal-footer** edit and view components).

Feel free to cherry-pick my commit, if you think it makes sense.

Reply:
👍